scripts/swagger/swag.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+
+###################################################################
+# Script Name	:   swag.sh                                                                                          
+# Description	:   Simple script to run swag.sh                                                                                                                                                                        
```

Comment:
yes it does but it just wraps different processes because I need to add a few information (security) on the generated json and yaml files. I also figured the `yq` won't be installed on most machines (`jq` is a requirement is we can assume that every developer has `jq` installed).

Reply:
ok, well in any case, maybe just update the description to be more accurate? the way it's written sounds like this script calls itself, which doesn't seem to make sense.

Reply:
Resolved on the latest commit.